jaxrs-tck/src/main/java/ee/jakarta/tck/ws/rs/ee/resource/java2entity/JAXRSClientIT.java

```diff
@@ -46,10 +46,14 @@ public class JAXRSClientIT extends JAXRSCommonClient {
   private static final long serialVersionUID = 1L;
 
   public JAXRSClientIT() {
-    setup();
     setContextRoot("/jaxrs_resource_java2entity_web/resource");
   }
 
+  @BeforeEach
+  public void setup() {
+    super.setup();
+  }
+
```

Comment:
This shouldn't be required.

Reply:
The parent setup() method in jaxrs-tck/src/main/java/ee/jakarta/tck/ws/rs/common/JAXRSCommonClient.java need to be invoked only for test classes that deploys archives and not others.

Reply:
Sorry, for some reason I thought it was also annotated with `@BeforeEach`. I must have looked at the wrong file.